chapters/operatorsandexpressions.tex

```diff
@@ -804,7 +804,7 @@ \subsubsection{spatialDistribution}\label{spatialdistribution}
 solver.
 \end{nonnormative}
 
-\lstinline!spatialDistribution! lets one approximate efficiently the solution of the infinite-dimensional problem
+\lstinline!spatialDistribution! allows the infinite-dimensional problem below to be solved efficiently with good accuracy
```

Comment:
Not bad at all.  How about some punctuation (like a period or a colon) at the end, before the equation?

Reply:
Hmm... I will have to think more about which punctuation is best, so take that later.